itests/hcatalog-unit/src/test/java/org/apache/hive/hcatalog/listener/TestDbNotificationListener.java

```diff
@@ -392,6 +399,8 @@ public void alterDatabase() throws Exception {
     String newDesc = "test database";
     Database dbAfter = dbBefore.deepCopy();
     dbAfter.setDescription(newDesc);
+    dbAfter.setOwnerName("test2");
```

Comment:
As alterDatabase operation will generate events only in case of change in dbOwner
https://github.com/apache/hive/blob/8ea1d78d0dc18fec4305dd0d0a9e341ecdcaef1e/standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/events/AlterDatabaseEvent.java#L79

Reply:
thx